lib/ecto/schema.ex

```diff
@@ -252,7 +252,7 @@ defmodule Ecto.Schema do
   defined so structs and changeset functionalities are available.
   """
 
-  @type t :: %{__struct__: atom}
```

Comment:
Ah, thanks. Should the docs be updated? https://github.com/elixir-lang/elixir/blob/bcdf0415d9589bc0a220c499715947270e917b9c/lib/elixir/pages/Typespecs.md#L114